lapce-app/src/app.rs

```diff
@@ -2219,7 +2220,7 @@ fn palette_item(
     palette_item_height: f64,
     config: ReadSignal<Arc<LapceConfig>>,
     keymap: Option<&KeyMap>,
-) -> impl View {
+) -> impl View + use<> {
```

Comment:
This use of `use<>` is the only syntax/semantic change of this PR. The rest of it is just cargo fmt reordering imports using case-sensitive order.